aws/resource_aws_organizations_organization.go

```diff
@@ -106,7 +142,18 @@ func resourceAwsOrganizationsOrganizationRead(d *schema.ResourceData, meta inter
 		return fmt.Errorf("error describing Organization: %s", err)
 	}
 
+	log.Printf("[INFO] Listing Roots for Organization: %s", d.Id())
+	var roots []*organizations.Root
+	err = conn.ListRootsPages(&organizations.ListRootsInput{}, func(page *organizations.ListRootsOutput, lastPage bool) bool {
+		roots = append(roots, page.Roots...)
+		return !lastPage
+	})
+	if err != nil {
+		return err
+	}
+
 	d.Set("arn", org.Organization.Arn)
+	d.Set("roots", flattenOrgRoots(roots))
```

Comment:
When using `d.Set()` with aggregate types (TypeList, TypeSet, TypeMap), [we should perform error checking to prevent issues where the code is not properly able to set the Terraform state](https://www.terraform.io/docs/extend/best-practices/detecting-drift.html#error-checking-aggregate-types). e.g.

```suggestion
	if err := d.Set("roots", flattenOrgRoots(roots)); err != nil {
		return fmt.Errorf("error setting roots: %s", err)
	}
```